portal/add_edit_event_user.php

```diff
@@ -487,8 +487,8 @@
     $user = sqlQueryNoLog("SELECT users.username FROM users WHERE authorized = 1 And id = ?", array($_POST['form_provider_ae']));
     $rtn = addPnote($_POST['form_pid'], $note, 1, 1, $title, $user['username'], '', 'New');
 
-    $_SESSION['whereto'] = 'appointmentcard';
-    header('Location:./home.php#appointmentpanel');
+    OpenEMR\Common\Session\SessionUtil::setSession('whereto', '#appointmentcard');
+    header('Location:./home.php');
```

Comment:
@adunsulag Here is the place most likely was your issue but went through and fixed everywhere in portal.